devicetypes/smartthings/smartweather-station-tile.src/smartweather-station-tile.groovy

```diff
@@ -347,11 +351,11 @@ private estimateLux(obs, sunriseDate, sunsetDate) {
     }
     else {
         //day
-        switch(obs.iconCode) {
-            case '04':
+        switch(obs.iconCode as String) {
+            case '4':
                 lux = 200
                 break
-            case ['05', '06', '07', '08', '09', '10',
+            case ['5', '6', '7', '8', '9', '10',
```

Comment:
I still need to test this, but should 16 be in the list too? Also, if we changed the cases to integers we could do `case 5..26:` which seems like a nice improvement

Reply:
16 looks like snow so it seems like it should be included. I switched to using integer ranges.